kornia/feature/adalam/utils.py

```diff
@@ -0,0 +1,133 @@
+import math
+
+import torch
+
+
+def arange_sequence(ranges):
+    """
+    returns a sequence of the ranges specified by the argument.
+    Example:
+    [2, 5, 1, 2] -> [0, 1, 0, 1, 2, 3, 4, 0, 0, 1]
+    """
+    maxcnt = torch.max(ranges).item()
+    numuni = ranges.shape[0]
+    complete_ranges = torch.arange(maxcnt, device=ranges.device).unsqueeze(0).expand(numuni, -1)
```

Comment:
Is there any way to sort out not used code ? To avoid having dead/untested code

Reply:
I think, it is all being used. It is also quite complex, so, not easy to refactor, unfortunately.